lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -37,6 +37,19 @@ using namespace mlir::torch;
 
 namespace {
 
+Type toTorchTensorTypeCast(PatternRewriter &rewriter, Type ty) {
+  auto tensorTy = ty.cast<TensorType>();
+
+  auto elementType = dyn_cast<IntegerType>(tensorTy.getElementType());
+  if (elementType && elementType.isSignlessInteger()) {
+    elementType = rewriter.getIntegerType(tensorTy.getElementType().getIntOrFloatBitWidth(), true);
+    return Torch::ValueTensorType::get(
+        ty.getContext(), tensorTy.getShape(), elementType);
+  }
+  return Torch::ValueTensorType::get(
```

Comment:
Do we need a special case for `i1`?

Reply:
Good point! I think so. Let me check what happens in that case, maybe that is already covered somehow and I can do with a comment as well.

Reply:
Fixed